### ISSUE TITLE
Emoji box hidden

### ISSUE THREAD

**Issue by Cloud-and-Proud** [Sav22999/emoji#94]
Hello. Sorry if this is a bad place to put this but I didn't know where else to report this issue.
I changed the settings on Emoji to show larger emojis in the selector, and now the selector box has disappeared, like the size of the box has become very thin. This problem is even on the settings tab, so I can't try and change anything to fix it. See attached pictures.
![2023-03-25 13_58_44-Window](https://user-images.githubusercontent.com/57964924/227718631-ba2454e0-ee76-48f5-ae15-76a6a8913f17.png)
![2023-03-25 13_59_01-Window](https://user-images.githubusercontent.com/57964924/227718632-404ac26d-dd61-4697-8cf3-25903f254f3d.png)

**Comment by Sav22999** [comment]
@Cloud-and-Proud Oh, this is very weird 🤔 

Try to reinstall the add-on, because I haven't any other users reported this issue. Let me know
In addition, please, be sure you have the latest version of Firefox, because maybe it's a Firefox bug which has been already fixed

**Comment by TwistedZero** [comment]
Same issue here, I'm using the latest Nightly version of Firefox. Issue persists after:

- Reinstalling the add-on
- Uninstalling the add-on, restarting firefox, then reinstalling the add-on
- Downgrading the add-on

**Comment by TwistedZero** [comment]
I have managed to open the popup in a normal browser tab where the issue persisted.

In this state I was able to use the "Inspect" option in the right click menu to expand the page and access the add-on settings where I noticed the `Columns` and `Rows` options were both blank. it is likely they are being interpreted as zero when blank.

I was able to change them to normal values which fixed this issue.


Here are some steps that reproduce this issue for me:

1. Set the emoji size to the default (middle).
2. Set `Columns` and `Rows` options to the maximum value possible at that size.
3. Set the emoji size to Large.
4. The popup should now be tiny, and the `Columns` and `Rows` options should now be blank.
